packages/backend-common/src/logging/rootLogger.test.ts

```diff
@@ -14,18 +14,13 @@
  * limitations under the License.
  */
 
-import { PassThrough } from 'stream';
 import winston from 'winston';
 import { getRootLogger, setRootLogger } from './rootLogger';
 
 describe('rootLogger', () => {
   it('can replace the default logger', () => {
-    const logger = winston.createLogger({
-      transports: [
-        new winston.transports.Stream({ stream: new PassThrough() }),
-      ],
-    });
-    jest.spyOn(logger, 'info');
+    const logger = winston.createLogger();
+    jest.spyOn(logger, 'info').mockReturnValue(logger);
```

Comment:
I really don't wanna go here

Reply:
Heh. It works fine without, but it outputs warnings when running the tests if we try to log to a logger that has no transport :)